types/graphql/type/definition.d.ts

```diff
@@ -265,7 +269,7 @@ export interface GraphQLResolveInfo {
     variableValues: { [variableName: string]: any };
 }
 
-export type ResponsePath = { prev: ResponsePath, key: string | number } | void;
```

Comment:
Is there a reason why `| undefined` should be here instead of marking the places `ResponsePath` is used as optional?